### DISCUSSION TITLE
counting issue

### DISCUSSION THREAD

**Discussion by hyung000620** [DenverCoder1/github-readme-streak-stats#561] [Q&A]
I changed the repository once to private and then to public again... but the counting is getting weird. may i know what could be the cause?

**Comment by DenverCoder1** [comment]
The stats show whatever the GitHub API returns and images may be cached by GitHub's servers for up to a few hours.

This is what is shows when I look at your profile and everything seems to match the contribution graph:

![image](https://github.com/DenverCoder1/github-readme-streak-stats/assets/20955511/d0aae4dd-0560-48ed-851e-7e8dcb3cd7d0)

**Reply by Cfomodz** [reply]
@hyung000620 Make sure to mark this discussion as answered if it is :)
